lib/autoload/coursemology_docker_container.rb

```diff
@@ -40,7 +40,8 @@ def create(image, argv: nil)
                                               image: image) do |payload|
         options = { 'Image' => image }
         options['Cmd'] = argv if argv.present?
-        options['HostConfig'] = { 'memory' => CONTAINER_MEMORY_LIMIT, 'LogConfig' => LOG_CONFIG }
+        options['HostConfig'] = { 'memory': CONTAINER_MEMORY_LIMIT, 'LogConfig': LOG_CONFIG }
+        options['NetworkDisabled'] = true
```

Comment:
Should we store this boolean (default: true) with `course_assessment_question_programming` and read on container initialization?

Reply:
Let's do this only if someone really needs it. Else there'll be an additional boolean for every question and it could be used to do bad things to other servers.